apps/ideaspace/src/components/modules/EditIdea/EditIdea.js

```diff
@@ -86,6 +87,17 @@ function EditIdea() {
     }
   };
 
+    const compareValuesToInitial = (values) => {
+      const name = Object.keys(values);
```

Comment:
Just for readability, it would be better to name this variable `names`

Reply:
changed it to differWithInitial